MapleServer2/PacketHandlers/Game/StatPointHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using System;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class StatPointHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.STAT_POINT;
+
+        public StatPointHandler(ILogger<StatPointHandler> logger) : base(logger) { }
+
+        private enum StatPointMode : byte
+        {
+            Increment = 0x2,
+            Reset = 0x3
+        }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            StatPointMode mode = (StatPointMode) packet.ReadByte();
+
+            switch (mode)
+            {
+                case StatPointMode.Increment:
+                    HandleStatIncrement(session, packet);
+                    break;
+                case StatPointMode.Reset:
+                    HandleResetStatDistribution(session);
+                    break;
```

Comment:
Can you add 
```
default:
    IPacketHandler<GameSession>.LogUnknownMode(mode);
    break;
``` 
just like is seen inside `ClubHandler.cs`

Reply:
Added!